doc/stdlib/collections.rst

```diff
@@ -543,21 +543,21 @@ Iterable Collections
        julia> all(i->(4<=i<=6), [4,5,6])
        true
 
-.. function:: foreach(f, c...) -> Void
+.. function:: foreach(f, c...) → Void
 
    .. Docstring generated from Julia source
 
    Call function ``f`` on each element of iterable ``c``\ . For multiple iterable arguments, ``f`` is called elementwise. ``foreach`` should be used instead of ``map`` when the results of ``f`` are not needed, for example in ``foreach(println, array)``\ .
 
-.. function:: map(f, c...) -> collection
+.. function:: map(f, c...) → collection
 
    .. Docstring generated from Julia source
 
    Transform collection ``c`` by applying ``f`` to each element. For multiple collection arguments, apply ``f`` elementwise.
 
    .. doctest::
 
-       julia> map((x) -> x * 2, [1, 2, 3])
+       julia> map((x) → x * 2, [1, 2, 3])
```

Comment:
Thanks! it's my mistake. I corrected most of these cases before commiting, but I let this one pass.